dlhub_sdk/models/servables/python.py

```diff
@@ -113,14 +118,15 @@ class PythonClassMethodModel(BasePythonServableModel):
     any arguments of the class that should be set as defaults."""
 
     @classmethod
-    def create_model(cls, path, method, function_kwargs=None) -> 'PythonClassMethodModel':
+    def create_model(cls, path, method, function_kwargs=None, *, auto_inspect=False) -> 'PythonClassMethodModel':
```

Comment:
interesting use of `*` here, I hadn't been familiar with that before reviewing this code! learn something new every day